client/src/components/pages/CreateQuiz.js

```diff
@@ -0,0 +1,315 @@
+import React, { useState } from "react";
+import Navigation from '../layouts/Navigation'
+import {Button, Modal, Form, FormControl, InputGroup, Container} from 'react-bootstrap'
+import DynamicForm from '../utils/DynamicForm'
+import Questions from '../utils/Questions'
+
+const ColoredLine = ({ color }) => (
+  <hr
+      style={{
+          color: color,
+          backgroundColor: color,
+          height: 1,
+      }}
+  />
+);
+
+function CreateQuiz (props){
+
+    const [notes, setNotes] = useState([]);
+
+    //Temp data to hold edit data
+    const [tempNote, setTemp] = useState({
+      id:0,
+      Question: "",
+      Type:"NA",
```

Comment:
Great job in adding the type. I think it'll help identify the question the form should generate. I'll look into either leaving on front end or adding to db